site/source/_posts/2018/10/15/Version-4-Dojo.md

```diff
@@ -70,8 +70,6 @@ The existing widgets received many accessibility, keyboard, and focus refinement
 
 All of the breaking changes in Dojo 4, were carefully considered beforehand as we understand the impact that they can have on your decision to upgrade. To assist with the transition we have updated the CLI upgrade command, which will automatically upgrade your Dojo dependencies, upgrade your application code where possible and highlight areas in the application that require manual intervention. For more information on what has changed in Dojo 4, please see the [migration guide][Migration Guide].
 
-TypeScript forwards-compatibility was updated from 2.7 to 3.0.
```

Comment:
We should probably list what versions of TS that Doio 4 is known to work with. Otherwise this is fine.

Reply:
It’s only the version we ship with cli-Create-app... we should be better on the readmes I think also